go.mod

```diff
@@ -64,15 +65,16 @@ require (
 	golang.org/x/crypto v0.0.0-20200221231518-2aa609cf4a9d
 	golang.org/x/net v0.0.0-20200226121028-0de0cce0169b
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
-	golang.org/x/sys v0.0.0-20200327173247-9dae0f8f5775
+	golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
 	google.golang.org/genproto v0.0.0-20200227132054-3f1135a288c9
 	google.golang.org/grpc v1.27.1
 )
 
 replace (
-	github.com/containerd/containerd => github.com/containerd/containerd v1.4.0-beta.2.0.20200730150746-fa1220fce33f
 	github.com/docker/docker => github.com/docker/docker v17.12.0-ce-rc1.0.20200310163718-4634ce647cf2+incompatible
+	// protobuf: corresponds to containerd
+	github.com/golang/protobuf => github.com/golang/protobuf v1.3.5
```

Comment:
why is this needed now?

Reply:
libcontainer rc92 uses protobuf v1.4, but protobuf v1.4 has a breaking change: https://github.com/containerd/ttrpc/issues/62 (and also https://github.com/moby/buildkit/pull/1645#issuecomment-675043626)

BuildKit is probably unaffected, but it seems safer to use the version that corresponds to containerd.

Reply:
What a mess, but ok. Any chance we can get runc to not switch to version with a breaking change(fake v2 API), or do they actually use it for something?

Reply:
I'm rather inclined to reduce dependency on libcontainer

Reply:
👍 on reducing dependency on libcontainer where possible; there has been some discussion in the runc repository, where it was mentioned that libcontainer should not be considered a stable API, and not designed for external users, so if we can depend less on it, I think we should.

Reply:
stable go api is a lie anyway. Runc versions the binary, not the go library so that is expected. If the use is accidental I'm fine with refactoring but we shouldn't shy away or duplicate useful code.